test/unit/task_tests.rb

```diff
@@ -863,12 +863,18 @@ class CallbacksTask
     include Dk::Task
 
     before         CallbackTask, 'callback' => 'first_before'
+    before         CallbackTask, 'callback' => 'first_before'
+    before         CallbackTask, 'callback' => 'second_before'
     before         CallbackTask, 'callback' => 'second_before'
     prepend_before CallbackTask, 'callback' => 'prepend_before'
+    prepend_before CallbackTask, 'callback' => 'prepend_before'
 
     after         CallbackTask, 'callback' => 'first_after'
+    after         CallbackTask, 'callback' => 'first_after'
+    after         CallbackTask, 'callback' => 'second_after'
     after         CallbackTask, 'callback' => 'second_after'
     prepend_after CallbackTask, 'callback' => 'prepend_after'
+    prepend_after CallbackTask, 'callback' => 'prepend_after'
```

Comment:
@kellyredding - You might add comments to the duplicates here to explain that they are needed and not a typo/mistake and that they are testing that we don't run the callbacks multiple times (your order tests would break if they were run multiple times).